app/views/LocationShare/LiveLocationStatusBar.tsx

```
@@ -0,0 +1,161 @@
+import React, { useState, useEffect, useRef } from 'react';
+import {
+	View,
+	Text,
+	TouchableOpacity,
+	StyleSheet,
+	Animated,
+	InteractionManager,
+	Linking,
+	Platform,
```

ESLint

[error] 'Platform' is defined but never used.

(@typescript-eslint/no-unused-vars)